go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/confluentinc/confluent-kafka-go
+
+go 1.13
+
+require (
+	github.com/alecthomas/kingpin v2.2.6+incompatible
```

Comment:
These are not needed for the kafka/ module itself, only for the examples.
Can we put the go.mod in kafka/ instead?

Reply:
I'd make the examples their own module to separate them from the top-level module so their dependencies are separate. This is a perfect use case for it. If I was using the client's module and saw that it imported kingpin but didn't seem to use it, I wouldn't like it.

Reply:
Okay. I will do:

```
confluent-kafka-go/
    go.mod
    examples/
        go.mod
    kafkatest/
        go.mod
``` 

This way, the import path of `kafka` will stay the same `github.com/confluent-kafka-go/vX/kafka`, instead of changing to `github.com/confluent-kafka-go/kafka/vX` (what would happen if we put the go.mod in `kafka` instead of top-level.

Reply:
Is that correct though?
What we're interested in is the api version 1 of kafka/, so in my mind "github.com/confluent-kafka-go/kafka/v1" makes more sense, rather than the kafka api of version 1.

Reply:
As discussed offline, the module has the version, not the package. `github.com/confluent-kafka-go/vX/kafka` means "The `kafka` package of the `vX` version of the `confluent-kafka-go` module".